tests/autoyast/clone.pm

```diff
@@ -19,11 +19,18 @@
 use strict;
 use base 'console_yasttest';
 use testapi;
+use version_utils 'is_sle';
 
 sub run {
     my $self = shift;
     assert_script_run 'rm -f /root/autoinst.xml';
-    assert_script_run 'yast2 --ncurses clone_system', 400;
+    script_run('(yast2 --ncurses clone_system; echo yast2-clone_system-status-\$?) | tee /dev/$serialdev', 0);
```

Comment:
As you use single quotes we don't need to escape \$ for the last executed command exit code.

Reply:
I replaced the line with double quotes, simple change is better.:)  Verify log: http://openqa-apac1.suse.de/tests/3131#